COVIDWatch iOS/ViewControllers/Share.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  share.swift
+//  COVIDWatch iOS
+//
+//  Created by Laima Cernius-Ink on 4/8/20.
+//  Copyright © 2020 IZE. All rights reserved.
+//
+
+import UIKit
+
+class Share: BaseViewController {
+    var img: UIImageView?
+    var largeText = LargeText(text: "Share & Protect")
+    var mainText = MainText(text: "Covid Watch is using bluetooth to anonymously watch who you come in contact with. You will be notified of potential contact to COVID-19.")
+    var spreadButton = Button(text: "Spread the word", subtext: "It works best when everyone uses it.")
+    var testedButton = Button(text: "Tested for COVID-19?", subtext: "Share your result anonymously to help keep your community stay safe.")
+    let screenSize: CGRect = UIScreen.main.bounds
+    var scalingFactor: CGFloat?
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.scalingFactor = CGFloat(screenHeight) / CGFloat(896)
+        let imageb: UIImage = UIImage(named: "woman-hero-blue-2")!
+        img = UIImageView(image: imageb)
+        img!.frame.size.width = 253
+        img!.frame.size.height = 259
+
+        img!.center.x = view.center.x - 5
+        img!.center.y = 240
+        if screenHeight <= 736.0 { img!.center.y *= scalingFactor! }
```

Comment:
Can you use `if let` syntax for this instead of force unwrapping?

Reply:
Yeah I'm going to revise this whole file in the next commit so will just ignore this for now